src/main/kotlin/uk/gov/justice/digital/hmpps/approvedpremisesapi/util/PaginationUtils.kt

```diff
@@ -47,34 +64,33 @@ fun getPageable(sortBy: String, sortDirection: SortDirection?, page: Int?, pageS
   }
 }
 
-@Deprecated("This function will ignore sorting if no page is defined. This is most likely not the expected behaviour", ReplaceWith("getPageableOrAllPages"))
-fun <SortType> PageCriteria<SortType>.toPageable(sortByConverter: (SortType) -> String) = getPageable(
-  sortByConverter(this.sortBy),
-  this.sortDirection,
-  this.page,
-  this.perPage,
-)
-
-fun <SortType> PageCriteria<SortType>.toPageableOrAllPages(unsafe: Boolean = false, sortByConverter: (SortType) -> String) = getPageableOrAllPages(
-  sortByConverter(this.sortBy),
-  this.sortDirection,
-  this.page,
-  this.perPage,
-  unsafe = unsafe,
-)
-
 fun getPageableOrAllPages(sortBy: String, sortDirection: SortDirection?, page: Int?, pageSize: Int?, unsafe: Boolean = false): Pageable {
+  return getPageableOrAllPages(listOf(sortBy), sortDirection, page, pageSize, unsafe)
+}
+
+fun getPageableOrAllPages(sortBy: List<String>, sortDirection: SortDirection?, page: Int?, pageSize: Int?, unsafe: Boolean = false): Pageable {
```

Comment:
I am not sure if we need to use unsafe as it only used in tests

Reply:
It's used in main here - https://github.com/ministryofjustice/hmpps-approved-premises-api/blob/e84d0215581bfd831640e63c448cd10975811ddf/src/test/kotlin/uk/gov/justice/digital/hmpps/approvedpremisesapi/unit/service/cas1/Cas1OutOfServiceBedServiceTest.kt#L575

Reply:
Changed it to use unsafe but I still not have clear why we need unsafe

Reply:
i believe it's because of the sort formula used here - https://github.com/ministryofjustice/hmpps-approved-premises-api/blob/e84d0215581bfd831640e63c448cd10975811ddf/src/test/kotlin/uk/gov/justice/digital/hmpps/approvedpremisesapi/unit/service/cas1/Cas1OutOfServiceBedServiceTest.kt#L567C27-L567C54

I agree, we should get rid of it if possible